### DISCUSSION TITLE
error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/

### DISCUSSION THREAD

**Discussion by eeriemyxi** [rushter/selectolax#52] [General]
I'm using Python 3.10. It works fine with 3.9 but not with 3.10. Any workarounds?

**Comment by rushter** [comment]
Have you tried installing the build tools? Python 3.10 is pretty new, so you need to compile the library.

**Reply by eeriemyxi** [reply]
I run a laptop that is comparable to a toaster so I cant install the build tools. I think I have to use 3.9 for this. I hope someone compiles it soon

**Reply by rushter** [reply]
Python 3.10 will be supported in the next release.

**Reply by eeriemyxi** [reply]
> Python 3.10 will be supported in the next release.

thank you

**Reply by eeriemyxi** [reply]
> 
> 
> Python 3.10 will be supported in the next release.

Hello, https://pypi.org/project/selectolax/0.3.6/, says 404, So I tried, 
```
pip install https://github.com/rushter/selectolax/archive/refs/tags/v0.3.6.zip
```
But I'm still getting [those errors](https://mystb.in/MoneyExperienceTent.coffeescript)

# Update
I checked again after 9+ hours, now the wheel is available. I installed it successfully. Thank you.
